src/War3Map.Template.Generated/War3Map.Template.Generated.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="War3Api.Common" Version="1.31.1.3" />
+    <PackageReference Include="War3Api.Common" Version="1.32.9" />
```

Comment:
I would prefer if this were changed to
```
<PackageReference Include="War3Api.Common.Legacy" Version="1.31.1.5" />
```
so it remains compatible with patch 1.31